src/rosdep2/main.py

```diff
@@ -123,16 +123,21 @@ class UsageError(Exception):
 """
 
 
-def _get_default_RosdepLookup(options):
+_global_options = None
+
+def _get_default_RosdepLookup(options=None):
```

Comment:
I think it is better to keep this
```suggestion
def _get_default_RosdepLookup(options):
```